build.py

```diff
@@ -290,15 +290,10 @@ def cmake(self, args):
         self.cmd(f'cmake {" ".join(env_args)} {" ".join(args)}', check_exitcode=True)
 
     def makeinstall(self, target="install"):
-        if target_platform() == "windows":
-            verbose_flag = "" if self._verbose else "-clp:ErrorsOnly"
-            self.cmd(
-                f"msbuild.exe -m:{FLAGS.build_parallel} {verbose_flag} -p:Configuration={FLAGS.build_type} {target}.vcxproj",
-                check_exitcode=True,
-            )
-        else:
-            verbose_flag = "VERBOSE=1" if self._verbose else "VERBOSE=0"
-            self.cmd(f"make -j{FLAGS.build_parallel} {verbose_flag} {target}")
+        verbose_flag = "-v" if self._verbose else ""
+        self.cmd(
+            f"cmake --build . --config {FLAGS.build_type} -j{FLAGS.build_parallel} {verbose_flag} -t {target}"
```

Comment:
Aren't build directory needs to be declared similar to other flags?

Reply:
could you elaborate? I don't see a build directory variable before these changes.

Reply:
In our scenarios we use `.` reference as CMake-generated project binary tree to run`cmake --build <dir>` .
I'm aware that  outcome of `build.py` will be a generated script which will create the directory for CMake-generated project binary tree and switch the location to it during the execution. 

However from the glance on a given line I see that the all argument values are given as a variables except CMake-generated project binary tree. 
Defined function by itself doesn't explains why `.` location is used and it's creates a confusion for those who not familiar yet with results. 
```python
f"cmake --build . --config {FLAGS.build_type} -j{FLAGS.build_parallel} {verbose_flag} -t {target}"
```

Reply:
what is the action here? Does adding a comment help clarify why "." is used?